pages/qci_ui_tools.py

```diff
@@ -0,0 +1,44 @@
+from selenium.webdriver.common.by import By
+
+
+class QciUiTools():
```

Comment:
An offline discussion (held online via IRC) resulted in this decision:

https://projects.engineering.redhat.com/browse/RHCIQE-318